src/targets/gpu/fuse_ops.cpp

```diff
@@ -778,7 +732,7 @@ void fuse_ops::apply(module& p) const
                         find_add_unary{"gpu::tanh", hip_add_tanh{}, hip_triadd_tanh{}},
                         find_add_clip{});
     run_passes(p, {dead_code_elimination{}});
-    match::find_matches(p, find_triadd_layernorm{}, find_gemm_add{}, find_commutative_broadcast{});
```

Comment:
`find_gemm_add` fusion should not be removed. It needs to be updated to use `alpha` and `beta` in the gemm op instead of using the dot operator.